lib/php/test/Unit/BaseValidatorTest.php

```diff
@@ -114,27 +120,35 @@ protected static function assertHasReadValidator($class)
     {
         if (!static::hasReadValidator($class)) {
             static::fail($class . ' class should have a read validator');
+        } else {
+            static::assertTrue(true);
         }
     }
 
     protected static function assertNoReadValidator($class)
     {
         if (static::hasReadValidator($class)) {
             static::fail($class . ' class should not have a write validator');
+        } else {
+            static::assertTrue(true);
         }
```

Comment:
What's the value of this? A check that is known to always succeed?

Reply:
if this check will be absent we will receive a warning that test does not make any assert, so if everything fine we should get a success assert